contracts/StatusMap.sol

```diff
@@ -38,10 +38,17 @@ contract StatusMap {
     /**
      * @dev Sets multiple bits for the account `account` using a provided `indicesMask`.
      */
-    function _setMultipleStatuses(address account, uint256 indicesMask) internal {
+    function _updateMultipleStatuses(address account, uint256 indicesMask) internal {
         statusesByAccount[account] |= indicesMask;
     }
 
+    /**
+     * @dev Sets multiple bits for the account `account` using a provided `indicesMask`.
+     */
+    function _setMultipleStatuses(address account, uint256 indicesMask) internal {
+        statusesByAccount[account] = indicesMask;
+    }
```

Comment:
In the context of this contract, a bit is set if it's equal to 1 and unset if it's 0.
So I wouldn't rename "_setMultipleStatuses" but name the new function "overwriteStatuses()"

Reply:
@0xpApaSmURf we have previously discussed about this renaming, would you be fine if I move forward with @ra-phael suggesiton?

Reply:
I think this a preference thing. Personally "set" reads like a strict write operation. It sets statuses there regardless and knows nothing else. "overwrite" implies writing over existing data and the verbiage is less simple IMO.

Adding "update" to the original function seems also more appropriate as the previous confusion I had regarding the function mostly derived from it being called "_setMultipleStatuses" when it is in fact an update for new statuses only.

Again, strong opinion from me but happy to weakly hold it in face of either of your choices 😄

Reply:
I agree with @0xpApaSmURf and we actually spent some time thinking about this yesterday, I believe the new naming convention is more descriptive, but also weakly held... Only con IMHO is the current cronjob being developed, but afaik it does not use the previous function

Reply:
I'm fine with changing the terminology used in StatusMap but then we should do it for all the other functions as well to be consistent throughout this contract. Because at the moment it is, if you look at the function one by one keeping in mind that: set = "setting a bit (or several) to 1" and unset = "setting a bit (or several) to 0". The only exception is `_setStatusTo()` which should be renamed `_updateStatusTo()` IMO since the value can be either 0 or 1.

Reply:
Ah I see, the "set" terminology is very much applying to the bits. In that case I think for non-individual bit-setting operations and we shouldn't use the term 'set' for a status combination.

If we wanted to be very strict about naming for the StatusMap contract (because its just a low-level bitmap manager), maybe the naming should be `setBit` when dealing with single index values (one status), and `setBits` when dealing with multiple index values (status combination).

By symmetry we end up with `unsetBit` and `unsetBits` for respectively removing statuses.

And then by extension we get your suggestion of `overwriteBit` and `overwriteBits` for singular and combination overwriting of statuses.

To me its very simple and elegant but I hope I haven't over-complicated it @ra-phael @sudoFerraz

Reply:
Then we reserve the usage of function naming that includes `update/grant/revoke` `status(es)` only to the VioletID contract that inherits